src/snowflake/snowpark/modin/plugin/extensions/base_overrides.py

```diff
@@ -97,7 +93,6 @@ def register_base_override(method_name: str):
     """
 
     def decorator(base_method: Any):
-        base_method = try_add_telemetry_to_attribute(method_name, base_method)
```

Comment:
do you need that decorator at all?

Reply:
do you mean the `register_base_override` annotation? we still need this to call setattr on BasePandasDataset + register_series_accessor/register_dataframe_accessor if necessary